src/components/ViewDeck/ViewDeck.js

```diff
@@ -97,228 +80,205 @@ class ViewDeck extends Component {
       Green: 0,
       Gray: 0,
     };
-    //.reduce
+    //TODO: .reduce instead
```

Comment:
I would like this TODO to be in this PR. Really focus on this one component before moving on. I still feel that you are a little all over the place with work on this app. Take one file at a time and make it the best it can be before moving on. That way you don't need to continue revisiting code until there are bug fixes and new features.